.gitignore

```diff
@@ -59,6 +59,9 @@ coverage.xml
 .pytest_cache/
 test.nc
 
+#test data generator
+tests/load/nwp/test_data_generator.py
+
```

Comment:
is this needed?

Reply:
@glitch401?

Reply:
looked over this, my bad
I dont think this is needed, as the test data is being [generated here](https://github.com/openclimatefix/ocf_datapipes/pull/340/files#diff-86f93ae3cac3067a2306efaa71403d4bb306b8b829a2b8cd5ef2f613c222a09aR114)